### PR TITLE
add figure schema in docs

### PR DESCRIPTION
**Pull request by atrisovic**
### All Submissions

**Follow best practices**

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/gdcc/pyDataverse/pulls) for the same update/change?
* [x] Have you followed the guidelines in our [Contribution Guide](https://pydataverse.readthedocs.io/en/master/contributing/contributing.html)?
* [x] Have you read the [Code of Conduct](https://github.com/gdcc/pyDataverse/blob/master/CODE_OF_CONDUCT.md)?
* [x] Do your changes in a seperate branch. Branches MUST have descriptive names.
* [x] Have you merged the latest changes from upstream to your branch?

**Testing**

* [x] Have you used tox and/or pytest for testing the changes?
* [x] Did the local testing ran successfully?

**Commits**

* [x] Have descriptive commit messages with a short title (first line).

**Describe the PR**

* Adds figure schema to make the explanations more user-friendly. 
* See screen shot:

Screen Shot 2021-02-16 at 5 11 39 PM

### CODE REVIEW

File: src/pyDataverse/docs/source/user/basic-usage.rst [hunk 1]
```diff
@@ -361,6 +371,51 @@ Dataset) attached to it.
 
 Now the Dataverse instance is as it was once before we started.
 
+.. _user_basic-usage_download-data:
```
Comment (skasberger) [inline src/pyDataverse/docs/source/user/basic-usage.rst:374]: I think this section should be earlier - after the "Upload Datafile" and before the "Retrieve all created data as a Dataverse tree" section.

File: src/pyDataverse/docs/source/user/basic-usage.rst [hunk 2]
```diff
@@ -72,10 +72,20 @@ can then be used (e. g. :meth:`json() <requests.Response.json>`).
 Create Dataverse Collection
 -----------------------------
 
-The top-level data-type in the Dataverse software is called a Dataverse collection, so we will
-start with that.
+The top-level data-type in the Dataverse software is called a Dataverse collection, so we will start with that. 
+Take a look at the figure below to better understand the relationship between a Dataverse collection, a dataset, and a datafile. 
 
-First, instantiate a :class:`Dataverse <pyDataverse.models.Dataverse>`
+.. figure:: ../_images/collection_dataset.png
```
Comment (skasberger) [inline src/pyDataverse/docs/source/user/basic-usage.rst:78]: It is a great idea, to show the relations visually. Some thoughts: it's not obvious from the diagram, that a dataverse could also have several dataverse inside, or more than one dataset (as it is clear for the datafiles).

File: src/pyDataverse/docs/source/user/basic-usage.rst [hunk 3]
```diff
@@ -72,10 +72,20 @@ can then be used (e. g. :meth:`json() <requests.Response.json>`).
...
+   :align: center
+   :alt: collection dataset datafile
+
+A dataverse collection (also known as a :class:`Dataverse <pyDataverse.models.Dataverse>`) acts as a container for your :class:`Datasets<pyDataverse.models.Dataverse>`.
```
Comment (skasberger) [inline src/pyDataverse/docs/source/user/basic-usage.rst:82]: It is a great idea for beginners to share a bit of the context before getting started.  Such as what the data-structure is.

File: src/pyDataverse/docs/source/user/basic-usage.rst (hunk 1 above)
Comment (atrisovic) [reply src/pyDataverse/docs/source/user/basic-usage.rst]: Ok changed!

**Comment by atrisovic** [comment]
I incorporated all of the suggestions

**Review by skasberger — APPROVED** [review]
ok.

**Pull request merged**